client/my-sites/activity/activity-log-item/style.scss

```diff
@@ -81,7 +81,7 @@
 }
 
 .activity-log-item__time {
-	font-size: 11px;
```

Comment:
@sixhours Would appreciate a review request to @Automattic/jetpack-design in changes to any Jetpack products, as this is breaking the layout as reported in p1HpG7-9xQ-p2#comment-40285

This would have given us the opportunity to make the code more robust to better support the font-size change here.

Reply:
Hey hey! Sure, I'm happy to /cc Jetpack Design on future typography adjustments. I was under the impression code owners would get cc'ed automatically, but I can see that didn't happen with Jetpack for any of these; maybe that only applies to JSX files. Thanks for turning around a quick fix!